src/components/Hero/Hero.mdx

```diff
@@ -198,6 +201,55 @@ subHeaderText={otherSubHeaderText}
 
 <Canvas of={HeroStories.CampaignBackgroundColors} />
 
+## Fallback Campaign Image
+
+The `imageProps` prop can be used to set a fallback for the **main** image in
+the `"campaign"` variant. The internal `Image` component will use the `onerror`
+event to load the `fallbackSrc` image.
+
+The `Hero` component does not use the `Image` component, however, for the
+background image. The background image is rendered through the CSS
+`background-image` property. This means that more work is needed to set a
+fallback for the background image.
+
+The following example demonstrates how to set a fallback image for both the main
+image and the background image. Note: this example assumes that both the main
```

Comment:
re: "background-image" not having a fallback image prop - would this only be a issue when the main and background image sources are different?

Reply:
Yes, that's right. Luckily (?), for DCF the same image is used for the main and background image.

This is a problem I will revisit later for this "campaign" variant and the "primary" variant.